common/lib/nft/storage/knft-ipfs.ts

```diff
@@ -22,7 +22,7 @@ const nftIpfsJson = async (
   ipfs = "",
   address = "",
   image = "",
-  properties: properties = {},
+  properties: Properties = {},
```

Comment:
put new param in last position , i.e. properties after metadata

Reply:
line 39   `if (Object.keys(properties)?.length > 0) json.properties = properties;` =>  `if (Object.keys(properties).length > 0) json.properties = properties;`